Tasks/Common/VstsAzureHelpers_/InitializeFunctions.ps1

```diff
@@ -150,18 +169,49 @@ function Set-CurrentAzureRMSubscription {
     $null = Select-AzureRMSubscription -SubscriptionId $SubscriptionId @additional
 }
 
-function Format-Splat {
+function Set-UserAgent {
```

Comment:
Currently there is no azure cmdlet to set this. I will start a parallel thread with them to see if they can expose but in the meantime we want to start getting numbers of azure deployment so once they expose we can change it. With the current implementation if the type doesn't exist than we just don't add the useragent and the behaviour will be same as before.

Rest of the comments I have resolved